components/contribution-flow/index.js

```diff
@@ -333,7 +333,8 @@ class CreateOrderPage extends React.Component {
 
   /** Validate step profile, create new org if necessary */
   validateStepProfile = async () => {
-    if (!this.state.stepProfile || !this.activeFormRef.current || !this.activeFormRef.current.reportValidity()) {
+    // We don't want to crash if `reportValidity` is not supported by the browser
+    if (!this.state.stepProfile || !this.activeFormRef.current || !reportValidityHTML5(this.activeFormRef.current)) {
```

Comment:
Thanks to the new helper, we can simplify this line

```suggestion
    if (!this.state.stepProfile || !reportValidityHTML5(this.activeFormRef.current)) {
```